sdv/tabular/copulagan.py

```diff
@@ -209,11 +212,13 @@ def _fit(self, table_data):
         fields = self._metadata.get_fields()
         transformers = {
             field: GaussianCopulaTransformer(
-                distribution=distributions.get(field, default)
+                distribution=distributions.get(field.replace('.value', ''), default)
             )
             for field in table_data
-            if field.replace('.value', '')
-            in fields and fields.get(field, dict()).get('type') != 'categorical'
+            if field.replace('.value', '') in fields and fields.get(
+                field.replace('.value', ''),
+                dict(),
+            ).get('type') != 'categorical'
```

Comment:
minor: but can we rewrite this using a normal for loop and storing `field.replace('.value', '')` as a variable. I think it would be easier to read